extensions/2.0/Vendor/PTC_onshape_metadata/schema/gltf.PTC_onshape_metadata.schema.json

```diff
@@ -5,6 +5,16 @@
     "description": "glTF extension for specifying IDs and visibility.",
     "allOf": [ { "$ref": "glTFProperty.schema.json" } ],
     "properties": {
+        "documentId" : {
+            "type" : "string",
+            "minLength": 1,
+            "description" : "An identifier of the document this object relates to."
+        },
+        "elementId" : {
```

Comment:
The minimum documentId/elementId size Onshape uses is 24, the specification currently has it set to 1. Is this by design?